test/integration/policy_role_test.go

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package integration
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	policiesv1 "github.com/stolostron/governance-policy-propagator/api/v1"
+	"github.com/stolostron/governance-policy-propagator/test/utils"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8stypes "k8s.io/apimachinery/pkg/types"
+
+	"github.com/stolostron/governance-policy-framework/test/common"
+)
+
+var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the policy-role policy", Label("policy-collection", "stable"), func() {
+	const (
+		policyRoleName   = "policy-role"
+		policyRoleURL    = policyCollectACURL + policyRoleName + ".yaml"
+		policyRoleNSName = policyRoleName + "ns"
+		roleName         = "sample-role"
+	)
```

Comment:
This is good. I was thinking the other test files shouldn't have package-level variables for their names.

Reply:
yep